src/components/Login/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import { firebaseConnect } from 'react-redux-firebase'
+import GoogleButton from 'react-google-button'
+import * as routes from '../../constants/routes';
+
+class Login extends Component {
+  onAuth = () => {
+    const {
+      history,
+      firebase,
+    } = this.props;
+
+    firebase.login({ provider: 'google', type: 'popup' })
+      .then(() => {
+        history.push(routes.HOME);
+      })
+      .catch(error => {
+        console.log("ERR", error);
+      });
+  }
+
+  render() {
+    const {
+      auth,
+    } = this.props;
+
+    return (
+      <div>
```

Comment:
I'm neither for or against this "dropped attributes style" *but* this is the only place you do it, so don't drop for consistency with the rest of the change.